packages/api/endpoints/granules.js

```diff
@@ -578,11 +579,15 @@ async function bulkOperations(req, res) {
  */
 async function bulkDelete(req, res) {
   const payload = req.body;
-
   if (payload.forceRemoveFromCmr && !isBoolean(payload.forceRemoveFromCmr)) {
     return res.boom.badRequest('forceRemoveFromCmr must be a boolean value');
   }
 
+  payload.concurrency = payload.concurrency ? Number(payload.concurrency) : 10;
+  payload.maxDbConnections = payload.MaxDbConnections
+    ? Number(payload.MaxDbConnections)
```

Comment:
Should we check that it's a valid integer after casting it as a number?

Reply:
Hmm.    In reality we should probably be doing a full validation on the input payload.        For the moment, how about that/fixing the units:

213e8342c4